### PR TITLE
Small changeling nerfs

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request

- increases changeling regen time
- changelings no longer spawn as teams
- hivemind chat is now disabled- no more getting ganged up on by lings
- resonant shriek cost increased to two. it's a temporary change for something new as a whole
- slightly changes absorb objective's text to not encourage murderboning
- increases spider cost and removes absorb req to not encourage murderboning
## Why It's Good For The Game

- team objectives are broken, and are unfun for all parties even when they work
- team changelings mean the antags automatically team up. it can be fun if there is an absorb ling, but it's otherwise a curbstomp if the changelings communicate and work together. Hivemind does this as well- eliminating the automatic, unstoppable communication already makes lings far easier to handle
- regenerative stasis was only forty seconds. increased to two minutes- this makes it less useful to get out of sticky situations, and indirectly buffs Last Resort. 
- shriek's cost was increased to make it take a bit more investment.
- changed some things to discourage random absorptions
## Changelog
:cl:
del: disables ling team objectives and hivemind
tweak: changed absorb objective text
tweak: changed spiders to not require absorption, but increased cost
balance: increased resonant shriek cost
balance: increased time needed to regen
/:cl:

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
I think nerfing reviving stasis is honestly cringe, (If you let the ling get back up and escape after death, that's on you) but everything else looks good.

**Comment by ATH1909** [comment]
> hivemind chat is now disabled- no more getting ganged up on by lings

cringe

also, you forgot to disable that ling power that lets you give someone temporary access to hivemind chat

File: code/game/gamemodes/changeling/changeling.dm [hunk 1]
```diff
@@ -75,7 +75,7 @@ GLOBAL_VAR(changeling_team_objective_type) //If this is not null, we hand our th
 	for(var/datum/mind/changeling in changelings)
 		log_game("[key_name(changeling)] has been selected as a changeling")
 		var/datum/antagonist/changeling/new_antag = new()
-		new_antag.team_mode = TRUE
+		new_antag.team_mode = FALSE //Should stop team lings from spawning
```
Comment (ike709) [inline code/game/gamemodes/changeling/changeling.dm:78]: If you're removing team lings, you should remove all of the relevant code from this proc.

Comment (zeskorion) [reply code/game/gamemodes/changeling/changeling.dm:78]: I'd not call this a full removal, rather simply disabling teamlings. Some people, though they are a minority, do enjoy them, and a time may come in which team lings are actually fair and balanced.

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -895,7 +895,7 @@ GLOBAL_LIST_EMPTY(possible_items_special)
 /datum/objective/changeling_team_objective //Abstract type
 	martyr_compatible = 0	//Suicide is not teamwork!
 	explanation_text = "Changeling Friendship!"
-	var/min_lings = 3 //Minimum amount of lings for this team objective to be possible
+	var/min_lings = 100 //Minimum amount of lings for this team objective to be possible //set to 100 to stop teamlings just in case
```
Comment (qwertyquerty) [inline code/game/gamemodes/objective.dm:898]: This isn't a good solution.

**Comment by qwertyquerty** [comment]
If you're gonna remove teams, just remove teams. Don't kind of sort of messily remove teams.

**Comment by qwertyquerty** [comment]
@zeskorion ?

**Comment by zeskorion** [comment]
yeaaaah i was puttin that off, i think i'm gonna close this shit for now, and reopen it later, cuz i  know a bit more now and can code it betterer with betterer changes and such

**Pull request closed without merging**